network/robus_network/HAL/STM32G4/robus_hal.c

```diff
@@ -198,7 +198,7 @@ _CRITICAL void RobusHAL_SetRxState(uint8_t Enable)
 _CRITICAL void ROBUS_COM_IRQHANDLER()
 {
     // Reset timeout to it's default value
-    RobusHAL_ResetTimeout(DEFAULT_TIMEOUT);
+    RobusHAL_ResetTimeout(true, 0);
```

Comment:
In this case you want to enable the timer at DEFAULT_TIMEOUT. Instead of 0 you could use DEFAULT_TIMEOUT to improve readability.
(This would allow you to remove the boolean parameters, see other review comment)